### ISSUE TITLE
Button variant action

### ISSUE THREAD

**Issue by AlbyIsCoding** [Altinn/altinn-design-system#26]
This button will probably be redundant after #23 is finished. So we'll wait with this component until it is finished.

See the picture and study the functionality of the component which can be found in /Profile when logging into https://at22.altinn.cloud/ui/Authentication with user Remy. We need to make the component in the picture below. After talks with @hannekristin we've come to agreement on creating the style below.
![image](https://user-images.githubusercontent.com/31471142/172790139-f9347d09-e78c-4696-a8bf-5ddd62bc0b34.png)

### Description
Simple description of component

### Figma
_Link to correct placement in our Figma component library._
- [ ] Check if there are similar components/variants from Altinn 2, Altinn Studio or Altinn 3 that need to be considered. 
- [ ] Are there any WCAG issues with the component
- [ ] Component is created in Figma
- [ ] All relevant states are created
- [ ] Functionality/animation is described and prototyped 

### Tokens
- [ ] The size is described with tokens
- [ ] spacing is described with tokens
- [ ] The color of all states are described with tokens
- [ ] Border width or styling is described with tokens
- [ ] Text size are described with tokens

```
Replace with size tokens

Replace with padding

Replace with colors

Replace with border width

Replace with text
```

### SVG
- [ ] If there are used any svgs in this component is it available for the developer?

**Comment by mjulstein** [comment]
@Albertlarsen It might need to be implemented in app-frontend instead, as the button in design-system is a basic button.
Or it si a different button variant.

**Comment by haakemon** [comment]
@mjulstein No, we shouldnt introduce primitive components outside the component library for the design system.

If this is a button variant that is needed, we should include it here.

**Comment by Albertlarsen** [comment]
I talked to Lasse earlier today and thought I'd use the new work process right away. Please assign the Button back to me when it's ready to be worked wtih @mrosvik
